### DISCUSSION TITLE
Failed Installation of Libmesh on HPC Cluster

### DISCUSSION THREAD

**Discussion by delinzha** [idaholab/moose#23434] [Q&A Getting Started] [answered]
For some reason, I have to install an old version of MOOSE on the HPC cluster (git checkout 34fb8fa). However, I met an error when running update_and_rebuild_libmesh.sh. The error is **/usr/bin/ld: cannot find -ludev**. I've checked similar discussions #18868 to run the command git submodule sync in the Libmesh submodule. However, it did not work on my end. Modules I loaded on HPC clusters are:

module load gcc/9.2.0
module load python/2.7.16
module load openblas/0.3.7
module load cmake/3.15.4
module load libevent/2.1.8
module load numactl/2.0.12
module load openmpi/4.1.1-gdr

I attached a snapshot of the error I met below:
![image](https://user-images.githubusercontent.com/114359380/219107560-71f0eec9-c7ee-4702-842d-ff1edcb2246d.png)

I would appreciate it if any experts could help me have a look at it. Thank you in advance.

**Comment by GiudGiud** [comment, marked as answer]
Does this fix it?
https://forum.qt.io/topic/60107/cannot-find-ludev

**Reply by delinzha** [reply]
Thank you for your suggestion. I do not have permission to run sudo on the HPC cluster. I'll talk with the admin about it.
